quantum/split_common/matrix.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SPLIT_COMMON_MATRIX_H
+#define SPLIT_COMMON_MATRIX_H
+
+#include <common/matrix.h>
+
+#ifdef RGBLIGHT_ENABLE
+#   include "rgblight.h"
+#endif
+
+typedef struct _Serial_m2s_buffer_t {
+#ifdef BACKLIGHT_ENABLE
+    uint8_t backlight_level;
+#endif
+#if defined(RGBLIGHT_ENABLE) && defined(RGBLIGHT_SPLIT)
```

Comment:
Where is `RGBLIGHT_SPLIT` used?

Reply:
Through the serial, there is a keyboard that needs to transfer rgblight_config. Also, some keyboards do not need to transfer rgblight_config through serial. I thought so, so I wrote RGBLIGHT_SPLIT. But this is uncertain.

Reply:
Specifically, some boards like the helix (and derived boards) don't transport the rgb data over the TRRS jack, so it still needs a way to communicate between halves.   

Eg, this is for compatibility when the helix and crkbd (aka heli-dox) can be converted to the split common code.